examples/2d/bloom_2d.rs

```diff
@@ -210,18 +180,33 @@ fn update_bloom_settings(
                 bloom.scale.x += dt * 2.0;
             }
             bloom.scale.x = bloom.scale.x.clamp(0.0, 16.0);
-
-            if keycode.just_pressed(KeyCode::KeyO) {
-                commands.entity(entity).insert(tonemapper.next());
-            }
         }
 
-        (entity, None) => {
-            text.0 = "Bloom: Off (Toggle: Space)".to_string();
+        (entity, _, None) => {
+            text.0 = "Bloom: Off (Toggle: Space)\n".to_string();
 
             if keycode.just_pressed(KeyCode::Space) {
                 commands.entity(entity).insert(Bloom::default());
             }
         }
     }
+
+    text.push_str(&format!("(O) Tonemapper: {:?}\n", bloom.1));
```

Comment:
Just a nit.

`bloom.1` is a little sad.

This whole thing would clean up well with:

```rust
    let (entity, tonemapping, maybe_bloom) = camera.into_inner();

    match maybe_bloom {
        Some(mut bloom) => {
```

At the top of the fn.